README.md

```diff
@@ -1,24 +1,28 @@
-# Homebrew
-Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).
+# **Homebrew**
+**Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).**
```

Comment:
I don't think the boldface is necessary here (it also has no effect on the header).